safaribooks.py

```diff
@@ -315,9 +311,9 @@ def __init__(self, args):
         self.display.intro()
 
         self.session = requests.Session()
-        if USE_PROXY:  # DEBUG
-            self.session.proxies = PROXIES
-            self.session.verify = False
+        if args_parsed.proxies:
```

Comment:
@lorenzodifuccia please check. I removed now completely the global arguments and packed it into some functions to avoid errors like this and make it more readable.